edx-olive/configuration-old
# SE-4105 fix cryptography installation failure

This fix pins `cryptography` to an older version to get builds working again.  This isn't ideal, since it risks missing out on security patches.  However, it seems like a reasonable temporary fix.

**Test instructions**:

- verify that ami's build successfully with this configuration branch. (See [SE-4105](https://tasks.opencraft.com/browse/SE-4105) for details.)

**Reviewers**:
